gdp-src-build/conf/templates/renesas-rcar-gen3.local.inc

```diff
@@ -0,0 +1,143 @@
+# Include generic GDP local configuration
+include local.inc
+
+# Add R-Car Gfx and MMP packages to install
+IMAGE_INSTALL_append_rcar-gen3 = "libdrm-kms kernel-module-gles gles-user-module libgbm-dev packagegroup-gstreamer1.0-plugins weston-bin kernel-module-pvrsrvkm"
+
+MACHINE ?= "m3ulcb"
+
+# This sets the SoC
+# H3: r8a7795, M3: r8a7796
+#SOC_FAMILY = "r8a7795"
+SOC_FAMILY ?= "r8a7796"
+
+# Add systemd configuration
+DISTRO_FEATURES_append = " systemd"
+VIRTUAL-RUNTIME_init_manager = "systemd"
+
+# Linaro GCC
+GCCVERSION = "linaro-5.2"
+
+PACKAGE_CLASSES = "package_ipk"
```

Comment:
No the YBSP can support either.

I selected ipk for two reasons. Firstly I thought of the recent request to move Genivi builds to be ipk only. Secondly there is an upstream issue in meta-linaro with gcc 5.2 when rpm is selected that Linaro seem to not want to fix. So selecting rpm requires a meta-linaro patch such as the one attached to workaround the issue:
[0001-libgcc-5.2-Workaround-upstream-build-failure-with-rp.patch.zip](https://github.com/GENIVI/genivi-dev-platform/files/807041/0001-libgcc-5.2-Workaround-upstream-build-failure-with-rp.patch.zip)

Given the first point I chose to solve the second by selecting ipk rather than carry a patch to a different layer.